### DISCUSSION TITLE
Add ability to export list of detected CVE from harbor web console

### DISCUSSION THREAD

**Discussion by ZrytyADHD** [goharbor/harbor#19107] [Ideas]
Hi,
I have an idea about exporting list of detected CVEs from Harbor.
Now when we need it, we have to use Harbor API. 
My idea is to add option to export whole list of CVE from Harbor web console to pdf,csv or html file.
This should help many peoples like me. In some company security team often require that kind of data in proper format html,csv or pdf.

Also great idea will be add option to export list of CVE from specific artifact, specific repository and whole project separated for each repository and artifact.

**Comment by chlins** [comment]
Replied in https://github.com/goharbor/harbor/issues/19112.
